### PR TITLE
return from test if on mac

### PR DESCRIPTION
**Pull request by Tixxx**

### CODE REVIEW

File: test/unittests/backends/webgl/test_pack_unpack.ts [hunk 1]
```diff
@@ -154,6 +154,15 @@ describe('#UnitTest# - pack - Tensor pack', () => {
     inferenceHandler = sessionhandler.createInferenceHandler();
   });
   const testDataSet = getTestData();
+
+  // TODO support WebGl 1.0 to run it on Mac
```
Comment (fs-eire) [inline test/unittests/backends/webgl/test_pack_unpack.ts:158]: use `createWebGLContext()` from webgl-context-factory.ts to perform a webgl version check

**Review by fs-eire — CHANGES_REQUESTED** [review]

File: test/unittests/backends/webgl/test_pack_unpack.ts (hunk 1 above)
Comment (Tixxx) [reply test/unittests/backends/webgl/test_pack_unpack.ts]: updated

File: test/test-runner.ts [hunk 2]
```diff
@@ -230,10 +230,42 @@ async function initializeSession(
 /**
  * run a single model test case. the inputs/outputs tensors should already been prepared.
  */
-export async function runModelTestSet(context: ModelTestContext, testCase: Test.ModelTestCase): Promise<void> {
-  Logger.verbose('TestRunner', `Start to run test data from folder: ${testCase.name}`);
+// TODO fix the reshape ops to be compatible with webgl1
```
Comment (xzhu1900) [inline test/test-runner.ts:233]: reshape and flatten ops?

**Review by xzhu1900 — APPROVED** [review]

File: test/test-runner.ts (hunk 2 above)
Comment (Tixxx) [reply test/test-runner.ts]: although flatten is implemented with reshape, it would be good to look into flatten as well for corner cases. updated.

**Review by xzhu1900 — APPROVED** [review]

**Review by fs-eire — APPROVED** [review]
